### ISSUE TITLE
Add {from,to}-jwt to mirror {from,to}-json

### ISSUE THREAD

**Issue by alexanderkjeldaas** [schibsted/jslt#22]
In some cases it's useful to encode and decode jwt tokens, or encrypted jwt tokens (jwe).

JWT is just authenticated (+ optionally encrypted) json, so having support for handling "secure" json and not only stringified would be useful.

**Comment by alexanderkjeldaas** [comment]
1) I think consuming or producing a single JWT is pretty common.  Having fields within json that consists of JWT tokens might be less common.

2) Can't be implemented without external dependencies, except by relying on DI.

**Comment by larsga** [comment]
Yes, I was thinking the same: we can make this as a module with a known name (like the experimental module that was introduced during the weekend) and have it as a separate artifact. I think it makes sense to do that in this repo. Basically move the core jslt down into one directory, and then make another one next to it for jslt-extras.

**Comment by pratikpparikh** [comment]
@larsga the service provider in my PR might help with separating core and extras out.
